roaster-tm4j/roaster-tm4j-common/src/main/java/hu/icellmobilsoft/roaster/tm4j/common/config/Tm4jReporterServerConfig.java

```diff
@@ -30,6 +35,18 @@
     private String password;
     private String basicAuthToken;
 
+    public String calculateUserName() {
```

Comment:
Nem jelzett hogy kellene doc erre? Elvileg be van mar allitva erre a projektre...